docs/reference/mapping/types/dense-vector.asciidoc

```diff
@@ -111,6 +111,40 @@ PUT my-index-2
 efficient kNN search. Like most kNN algorithms, HNSW is an approximate method
 that sacrifices result accuracy for improved speed.
 
+[[dense-vector-quantization]]
+==== Automatically quantize vectors for kNN search
+
+The `dense_vector` type supports quantization to reduce the memory footprint required when searching `float` vectors.
```

Comment:
"searching"  - this is specifically approx kNN, right? Not exact brute force search. If so, then maybe clarify this or just link to `knn-search.html#approximate-knn` ?